source/screens/App/screens/Home/screens/Home.css

```diff
@@ -0,0 +1,134 @@
+.userProfileContent {
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  min-width: 915px;
+  max-width: 1200px;
+}
+
+.topRow {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+}
+
+.profileIcons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.leadCount,
+.contractCount,
+.priceCount,
+.totalSummary {
+  width: 220px;
+  height: 70px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: #fff1e5;
+}
+
+.leadCount {
+  background-image: url(assets/images/lead-count.png);
```

Comment:
I'll check but you may actually need the strings with the quotes here.

Reply:
Yep, I do - I'll add them in